cpu/qn908x/periph/gpio.c

```diff
@@ -168,6 +171,7 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
 
 void gpio_irq_enable(gpio_t pin)
 {
+    NVIC_EnableIRQ(gpio_nvic_irqs[GPIO_T_PORT(pin)]);
```

Comment:
Nothing is disabling the NVIC port IRQ, you might also move this to `gpio_init_int()` to slightly speed `gpio_irq_disable()` / `gpio_irq_enable()` pairs.

Reply:
Done. Also fixed an unused warning when the `periph_gpio_irq` module is not used.